client/slack-spring-test-api-client/src/main/kotlin/io/olaph/slack/client/test/group/conversation/MockConversationMethodGroup.kt

```diff
@@ -46,7 +47,7 @@ class MockConversationMethodGroup : ConversationsMethodGroup {
     }
 
     override fun kick(authToken: String): ConversationsKickMethod {
```

Comment:
return type has to be as well MockConversationsKickMethod otherwise you would have to cast it manually

Reply:
ah yes, you're right - thank you for pointing that out!